interfaces/acados_matlab/ocp_create.c

```diff
@@ -844,9 +845,10 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
 	memcpy(fieldnames[4],"out",sizeof("out"));
 	memcpy(fieldnames[5],"solver",sizeof("solver"));
 	memcpy(fieldnames[6],"sens_out",sizeof("sens_out"));
+	memcpy(fieldnames[7],"plan",sizeof("plan"));
```

Comment:
Why do you need to keep the `plan`?

Reply:
I added it in order to be able to do checks depending on it.
It is good to remember what `plan` was chosen, when the object was created, since the user can change the other variables on the Matlab side.

Reply:
In the mex interface it is possible to do the same form the matlab structs, and it is done like that in some other mex file in the interface.

Either way works here.

Reply:
Checking against a matlab structs is just less save, because the user can modify it after `acados_sim` is created.